src/Main.java

```diff
@@ -1,4 +1,116 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");}
+        System.out.println("Hello world!");
+
+        System.out.println("Exercise 1");
+     byte bananas = 4;
+     System.out.println("Бананы"+ bananas + "кг");
+     int apples = 2500000;
+     System.out.println("Apples"+ apples +"kg");
+     short sugar = 30000;
+     System.out.println("Sugar"+ sugar +"kg");
+     long atom = 1000000000;
+     System.out.println("Atom in human"+ atom +"unity");
+     float water =  3.5F;
+     System.out.println("Bottle"+ water +"liters");
+     double bubble = 1.723F;
+     System.out.println("Bubble in soda"+ bubble +"unity");
+
+        System.out.println("Exercise 2");
+        float a = 27.12F;
+        System.out.println(a);
+        long b = 987678965549L;
+        System.out.println(b);
+        double c = 2.786;
+        System.out.println(c);
+        short d = 569;
+        System.out.println(d);
+        short i = -159;
+        System.out.println(i);
+        short f = 27897;
+        System.out.println(f);
+        byte g = 67;
+        System.out.println(g);
+        System.out.println("Exercise 3");
+
+        byte classA = 23;
+        byte classB = 27;
+        byte classC = 30;
+        short shortPaper = 480;
+        int students = classA + classB + classC;
+        int paperForStudents = shortPaper/students;
+        System.out.println("На каждого ученика рассчитано"+" " + paperForStudents+" "+ "листов бумаги");
+
+        System.out.println("Exercise 4");
+        byte bottle = 16;
+        byte time = 2;
+        int timePerBottles = bottle/time;
+        byte hour = 60;
+        int day = 24 * hour;
+        byte minute = 20;
+        int day3 = day*3;
+        int month = 30*day;
+        int bottlesIn20minutes = timePerBottles * minute;
+        int bottlesInDay = timePerBottles * day;
+        int bottlesIn3days = timePerBottles * day * day3;
```

Comment:
```suggestion
        int bottlesIn3days = timePerBottles * day3;
```
зачем умножать ещё на day?

Reply:
по невнимательности, в след. рад буду повнимательнее